pkg/controller/infrastructure/templates/main.tpl.tf

```diff
@@ -138,7 +153,10 @@ resource "aws_subnet" "nodes_z{{ $index }}" {
   vpc_id            = {{ $.vpc.id }}
   cidr_block        = "{{ $zone.worker }}"
   availability_zone = "{{ $zone.name }}"
-
+{{- if and $.enableDualstack $.create.vpc}}
+  ipv6_cidr_block = "${cidrsubnet(aws_vpc.vpc.ipv6_cidr_block, 8, (1 + ({{ $index }} * 3)))}"
```

Comment:
Similar to the flow-based approach, we do not use the first block, but start with offset 1. Why is that?